src/main/docker/Dockerfile

```diff
@@ -19,6 +19,7 @@ WORKDIR /build/cantaloupe
 COPY patches /build/cantaloupe-patches
 RUN if [ "${cantaloupe.version}" = "dev" ] ; then \
       git clone --quiet https://github.com/cantaloupe-project/cantaloupe.git . && \
+      git checkout -b 60c3e2bae4fa3458e1df29113172319fcd6102a3 60c3e2bae4fa3458e1df29113172319fcd6102a3;\
```

Comment:
I think it would be good to add a comment here, explaining why we're using a hash instead of the default branch. Could this problem also (alternatively) be solved by putting the hash in `cantaloupe.commit.ref`?

Reply:
I tried putting the hash in cantaloupe.commit.ref last week, and the security issues were still showing up. Tried it again this morning, and it seems like it's working now.